packages/node/test/integrations/linkederrors.test.ts

```diff
@@ -1,15 +1,26 @@
 import { ExtendedError } from '@sentry/types';
 import { createStackParser } from '@sentry/utils';
+import { NoopTransport } from '@sentry/core';
 
 import { Event, NodeClient } from '../../src';
 import { LinkedErrors } from '../../src/integrations/linkederrors';
 import { nodeStackParser } from '../../src/stack-parser';
 import { setupNodeTransport } from '../../src/transports';
+import { NodeClientOptions } from '../../src/types';
 
 const stackParser = createStackParser(nodeStackParser);
 
 let linkedErrors: any;
 
+function getDefaultNodeClientOptions(options: Partial<NodeClientOptions> = {}): NodeClientOptions {
```

Comment:
(optional) Because this function is the same in 3 test files, we could think about extracting it somewhere.